content/body/video-content.php

```diff
@@ -5,41 +5,41 @@
-    can't hear all of the audio. <strong><em>They are different from subtitles, which involve translating the video's
-        language into an alternate language &mdash;</em></strong> closed captions are in the same language as the audio.
+  <dd>Text that appears during the playing of a video, usually placed at the bottom of the viewing area, that displays what people in the video are saying in realtime. 
+    Captions also describe other important audio content in a video, like sound effects in the background. <strong><em>They are different from subtitles, which involve translating the video's
+    language into an alternate language &mdash;</em></strong> closed captions are in the same language as the audio.
     Subtitles don't usually contain any information about other non-spoken audio in a video (e.g. ambient sound, music,
     etc).</dd>
 
   <dt>Audio Descriptions:</dt>
-  <dd>An extra audio track with spoken commentary what help users with visual disabilities perceive content that is
+  <dd>An extra audio track with spoken commentary helps blind and partially-sighted users perceive content that is
     presented only visually. A narrator usually describes the visual-only content in the video. Audio descriptions can
     be implemented in several ways:
 
     <ol>
       <li>they can be included in the primary video</li>
       <li>they can be provided in a different audio track in the video. On broadcast television in North America, this
-        has be implemented using <a href="https://en.wikipedia.org/wiki/Second_audio_program">Secondary Audio
+        has been implemented using <a href="https://en.wikipedia.org/wiki/Second_audio_program">Secondary Audio
           Programming</a>.</li>
       <li>they can be recited by a computer voice speaking over the video content</li>
       <li>they can be implemented as an alternate version of the video that includes audio descriptions.</li>
     </ol>
 
-    <p><strong>Please note: If all of the visual information in described in the video's audio track already, then a separate audio description track is not necessary.</strong></p>
+    <p><strong>Please note: If all of the visual information is described in the video's audio track already, then a separate audio description track is not necessary.</strong></p>
   </dd>
 
   <dt>Transcripts:</dt>
-  <dd>A transcript is used by users who can neither hear audio nor see video, like deaf/blind users. It is like the
-    script the actors in the video use to know what is going on in the story that they are about to act in. A transcript
-    also should include descriptions of audio information in the video (like laughing) and visual information (such as
-    the creak of a door opening).
+  <dd>A transcript is used by users who can neither hear audio nor see video, like deaf or blind users. It is like the
+      script, the actors in the video used to know what was going on in the story that they were about to act in. A transcript
```

Comment:
I don't think the sentence makes sense when the comma is included:

```suggestion
      script the actors in the video used to know what was going on in the story that they were about to act in. A transcript
```